src/cc2olx/qti.py

```diff
@@ -516,7 +530,7 @@ def _mark_correct_responses(self, resprocessing, responses):
             for ans in correct_answers:
                 responses[ans.text]["correct"] = True
 
-            if respcondition.attrib["continue"] == "No":
+            if respcondition.attrib.get("continue", "No") == "No":
```

Comment:
Is it correct to set "No" as the default value?

Reply:
You can fincd in the [specification](https://www.imsglobal.org/profile/cc/ccv1p3/ccv1p3_qtiasiv1p2p1_v1p0.xsd) the next rows:
```
<xs:attribute name="continue" use="optional" default="No">
	<xs:simpleType>
		<xs:restriction base="xs:string">
			<xs:enumeration value="Yes"/>
			<xs:enumeration value="No"/>
		</xs:restriction>
	</xs:simpleType>
</xs:attribute>
```
It says that `continue` is `False` by default.